addons/smtp-mail.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+# T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
+
+# shellcheck disable=2034,2059
+true
+SCRIPT_NAME="SMTP Mail"
+SCRIPT_EXPLAINER="This script helps setting up an SMTP client for the OS, \
+that will be used to send emails about failed cronjob's and such."
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Must be root
+root_check
+
+# Show explainer
+explainer_popup
+
+# Check if smtp-mail was already configured
+print_text_in_color "$ICyan" "Checking if SMTP is already installed and configured..."
+if [ -f /etc/msmtprc ]
+then
+    # Ask for removal or reinstallation
+    reinstall_remove_menu
+    # Removal
+    apt-get purge msmtp -y
+    apt-get purge msmtp-mta -y
+    apt-get purge mailutils -y
+    apt autoremove -y
+    mv /etc/aliases.backup /etc/aliases
+    rm -f /etc/mail.rc
+    rm -f /etc/msmtprc
+    # Show successful uninstall if applicable
+    removal_popup
+else
+    print_text_in_color "$ICyan" "Installing smtp-mail..."
+fi
+
+# Install needed tools
+install_if_not msmtp
+install_if_not msmtp-mta
+install_if_not mailutils
+
+# Enter mailserver
+MAIL_SERVER=$(input_box_flow "Please enter the email server URL that you want to use.\nE.g. smtp.mail.com")
+
+# Enter if you want to use ssl
+while :
+do
+    PROTOCOL=$(input_box_flow "Please type in the encryption protocol for your mailserver.
```

Comment:
Please make this a whiptail with options instead. It's more user friendly.

Reply:
done